rxros2_cpp/README.md

```diff
@@ -45,19 +49,47 @@ This projects has received funding from the European Union's Horizon 2020 resear
 ## Dependencies
 The RxROS2 library depends on and uses the following software:<br>
 
-1. Ubuntu Bionic 18.04<br>
-2. ROS2 Eloquent Elusor<br>
-3. Reactive C++ v2<br>
-https://github.com/ReactiveX/RxCpp<br>
-Released under the Microsoft Open Source Code of Conduct.<br>
-The RxCpp library (header files) is installed as part of installing RxROS2<br>
+1. Ubuntu Focal Fossa 20.04 and ROS2 Foxy Fitzroy<br>
+2. Reactive C++ (RxCpp) v4.0<br>
+
+Please follow the installation instruction of Ubuntu at
+https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview
+
+and the installation instructions of ROS2 at
+https://index.ros.org/doc/ros2/Installation/.
```

Comment:
Just as with `rxros` (ie: v1), I'm not sure we need to mention this.

This is a ROS 2 package, so people need an OS where ROS 2 is supported and need to have it installed.

What we should definitely make clear is which *version* of ROS 2 is supported, *and* on which OS we have tested it.